manifests/dev/vsphere-7.0u2/guestcluster/1.17/pvcsi.yaml

```diff
@@ -362,8 +412,21 @@ data:
     [FeatureStatesConfig]
     name = "csi-feature-states"
     namespace = "{{ .PVCSINamespace }}"
+    [InternalFeatureStatesConfig]
+    name = "internal-feature-states.csi.vsphere.vmware.com"
+    namespace = "{{ .PVCSINamespace }}"
 kind: ConfigMap
 metadata:
   name: pvcsi-config
   namespace: {{ .PVCSINamespace }}
 ---
+apiVersion: v1
+data:
+  "volume-extend": "true"
+  "volume-health": "true"
+  "online-volume-extend": "false"
```

Comment:
How will the feature be tested by FVT if this is set to false ?

Reply:
They will have to set it to `true` before testing online resize.